tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderItemDeleteSteps.cs

```diff
@@ -10,14 +13,17 @@ internal sealed class OrderItemDeleteSteps
     {
         private readonly Request request;
         private readonly Settings settings;
+        private readonly OrderContext orderContext;
```

Comment:
Due to the final approach we don't need the order context so we can delete this line.

Reply:
Deleted this line.